package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "devDependencies": {
+ },
+  "scripts": {
+    "build": "node ./node_modules/gulp/bin/gulp.js",
+    "ensure-images": "node ./node_modules/gulp/bin/gulp.js ensure-images-optimized",
+    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/",
+    "serve-build": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve build/bundled/ -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/"
```

Comment:
We should also fix this in `lancie-frontend`, but this line is redundant. We can (apparently) already do `yarn run serve build/bundled`